src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/TestOutputLogger.cs

```diff
@@ -25,13 +25,25 @@ public ILogger CreateLogger(string categoryName)
             return _loggers.GetOrAdd(categoryName, (name, helper) => new TestOutputLogger(helper, name), _outputHelper);
         }
 
+        public ILogger<T> CreateLogger<T>()
```

Comment:
Does this extension method not work? https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loggerfactoryextensions.createlogger?view=dotnet-plat-ext-6.0#microsoft-extensions-logging-loggerfactoryextensions-createlogger-1(microsoft-extensions-logging-iloggerfactory)

Might need to add a `PackageReference` to `Microsoft.Extensions.Logging.Abstractions`.

Reply:
Sorry, I probably saw `ILogger*` and thought they were the same.